DerivedPriceFeedFlat.sol

```diff
@@ -6,40 +6,41 @@
 pragma solidity ^0.8.0;
```

Comment:
shouldn't be ^0.8.20?

Reply:
This file shouldn't exists in the first place, deleted it